### DISCUSSION TITLE
Blockquote Cite compatible with reactmarkdown?

### DISCUSSION THREAD

**Discussion by swijckmans** [remarkjs/remark#1175] [Q&A]
Hey folks, I've been breaking my head over getting the ["cite"](https://www.w3schools.com/tags/att_blockquote_cite.asp) attribute to work with react markdown and can not find a way. 
Any ideas or is this not possible today?

**Comment by ChristianMurphy** [comment]
Welcome @swijckmans! 👋 

Could you expand a bit on what you are asking?
See the support guide for some recommendations on how to ask questions so others can help: https://github.com/remarkjs/.github/blob/main/support.md

In particular some information which would help:
* What is your input markdown and expected output HTML?
* What are other solutions you have considered? (have you seen or considered https://github.com/remarkjs/remark-directive?)
